### ISSUE TITLE
open file from nextcloud not supported?

### ISSUE THREAD

**Issue by megacar1** [jhass/nextcloud-keeweb#248]
Hi,
Isn't it bizzare that keeweb for nextcloud only supports "Microsoft Teams", "Dropbox", "Google Drive", "OneDrive" but not bloody Nextcloud?
WebDav doesnt work - http error 401.

but why it cannot just open file from files of nextcloud?

EDIT: omg it works by navigating through files, clicking on the KDBX file haha

**Comment by arnowelzel** [comment]
Yes, the idea is not to open the KeeWeb app first and then select a file to open but to open the files from the Nextcloud "Files" app. Since the current situation with KeeWeb itself is still not totally clear, I am not sure how long updates for KeeWeb for Nextcloud will be provided. But removing the app icon may be a good idea to avoid that confusion.
